Userland/Libraries/LibPDF/Fonts/PDFFont.cpp

```diff
@@ -83,8 +84,7 @@ PDFErrorOr<NonnullRefPtr<PDFFont>> PDFFont::create(Document* document, NonnullRe
     if (subtype == "TrueType")
         return TRY(TrueTypeFont::create(document, dict, font_size));
 
-    dbgln("Unknown font subtype: {}", subtype);
-    TODO();
+    return Error(Error::Type::Internal, TRY(String::formatted("Unhandled font subtype: {}", subtype)).to_deprecated_string());
```

Comment:
Is this a good change? Previously we'd display the pdf still, and now we don't, right?

(There was a recent change to make this library more forgiving for invalid inputs, and this kind of counteracts that.)

Reply:
I think this is correct, as TODO() will crash the program . This way,  it *should* display this error message in the little text box instead.

Reply:
Thanks for explaining!